chapters/standard_subtypes/cameras.txt

```diff
@@ -80,7 +81,7 @@ below.
 [cols="<,<,>,<4",options="header,unbreakable"]
 |===================================================================================================
 | Name   | Type    |  Default | Description
-| fovy   |`FLOAT32`|   {pi}/3 | the field of view of the frame’s height
+| fovy   |`FLOAT32`|       60 | the field of view of the frame’s height specified in degrees
```

Comment:
We intentionally went with radians (as glTF camera does), see also e.g., the parameters of light sources. But that clarification is indeed missing here.

Reply:
Ok...I was looking at @jeffamstutz VisRTX implementation and thought the spec was off since he used degrees instead of radians. I'll change that back to pi/3 and state that it's specified in radians.